### PR TITLE
fix(OnyxSwitch): align placement of label and switch to figma + show marker on truncation

### PR DESCRIPTION
**Pull request by BoppLi**
Relates to #926 #904

- center align switch and label
- align click area with figma designs
- always show the optional/required marker on truncation
- align the skeleton design

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue [hunk 1]
```diff
@@ -216,6 +233,10 @@ $input-width: calc(
       }
     }
 
+    &__click-area {
+      padding: var(--onyx-switch-click-padding-vertical) var(--onyx-spacing-2xs);
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue:237]: TODO fix jumping of skeleton

File: packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-invalid--chromium-linux.png
Comment (larsrickert) [inline packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-invalid--chromium-linux.png:1]: Invalid state seems to be broken

File: packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-truncation-required-optional--chromium-linux.png
Comment (larsrickert) [inline packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-truncation-required-optional--chromium-linux.png:1]: Do you know why the second and third multiline examples have changed?

File: packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-webkit-linux.png
Comment (larsrickert) [inline packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-webkit-linux.png:1]: checked state is broken here

Comment (BoppLi) [reply packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-webkit-linux.png:1]: the "+" selector was the bad boy

**Comment by BoppLi** [comment]
> In Figma the switch height is:
> 
> * compact: 32px
> * default: 40px
> * cozy: 48px
> 
> Currently it is 34px for compact and 42px for default

fixed. I'll update the screenies

**Review by larsrickert — APPROVED** [review]

**Pull request merged**